js/data/locale/ku/dateformats.json

```diff
@@ -151,5 +157,172 @@
                 "s": "‏{sy} – {ey}"
             }
         }
-    }
+    },
+    "islamic": "gregorian",
```

Comment:
`cop/dateformats.json, ku/dateformats.json, prg/dateformats.json
cu/dateformats.json, vo/dateformats.json` files are still have duplicated contents from root

Reply:
Change the cldr tool to run `mergeAndPrune()` on the date formats before writing them out